src/main/kotlin/no/nav/familie/ef/sak/vilkår/MedlemskapMapper.kt

```diff
@@ -48,6 +48,8 @@ class MedlemskapMapper(
                     tilDato = it.tildato,
                     land = it.land?.let { land -> kodeverkService.hentLand(land, LocalDate.now()) },
                     årsak = it.årsakUtenlandsopphold,
+                    personidentUtland = it.personident,
```

Comment:
personidentUtland -> personidentEØSLand?  (ikke sååå viktig)

Reply:
Hvis disse verdiene kun gjelder for EØS så tenker jeg personIdentEØS og adresseEØS gir mening her og i kontrakter.

Reply:
Legger til EøsLand" suffikser